docs/noobaa.md

```diff
@@ -0,0 +1,45 @@
+# Noobaa Commands
+
+The Noobaa command supports all the noobaa cli arguments. See [noobaa cli](https://github.com/noobaa/noobaa-operator?tab=readme-ov-file#noobaa-cli)
+
+## Example
+
+```bash
+odf noobaa help
+```
+
+```bash
+Troubleshoot ODF noobaa
+
+Install:
+  install          Install the operator and create the noobaa system
+  upgrade          Upgrade the system, its components and CRDS
+  uninstall        Uninstall the operator and delete the system
+  status           Status of the operator and the system
```

Comment:
Could we show `odf noobaa status` as the example? Or something else simple, so we don't have so much output that could become stale soon.

Reply:
will do after testing